File: Level 70/Classwork/cw3.js
 
 
 function replace(s){
    let result = '';
    for(let i = 0; i < s.length; i++) {
      let char = s[i];
      if('aeiouAEIOU'.includes(char)) {
        result += '!';
      } else {
        result += char;
      }
    }
    return result;
  }

File: Level 42/Classwork/cw3.py
def replace_exclamation(sentence):
      vowels = "aeiouAEIOU"
      result = ""
      for char in sentence:
          if char in vowels:
              result += "!"
          else:
              result += char
      return result


text = "Hello World! How are you?"
print(replace_exclamation(text))

